invenio_app_ils/ui/src/invenio_app_ils/common/api/date.js

```diff
@@ -11,3 +11,7 @@ export const toISO = date => {
 export const toString = date => {
   return date.toLocaleString();
 };
+
+export const toShortDateTime = date => {
+  return fromISO(date).toFormat('yyyy-MM-dd HH:mm');
```

Comment:
I think that we should create a serializer for document and convert dates to the date object, use it internally and then transform it when displayed.
So this should become `date.toFormat('...')`

Reply:
done 🚀